docs/source/getting-started.md

```diff
@@ -0,0 +1,194 @@
+# Getting Started with ExecuTorch
+This section is intended to describe the necessary steps to take PyTorch model anda run it using ExecuTorch. To use the framework, you will need to take the following steps:
+- Install the ExecuTorch python package and runtime libraries.
+- Export the PyTorch model for the target hardware configuration.
+- Run the model using the ExecuTorch runtime APIs.
+
+<hr/>
+
+## Installation
+To use ExecuTorch, you will need to install both the Python package and the appropriate platform-specific runtime libraries.
+
+Pip is the recommended way to install the ExecuTorch python package. This package includes the dependencies needed to export a PyTorch model, as well as Python runtime bindings for model testing and evaluation.
+```
+pip install executorch
+```
+
+#### System Requirements
+- Python 3.10 - 3.12
+- g++ version 7 or higher, clang++ version 5 or higher, or another C++17-compatible toolchain.
+- Linux or MacOS operating system (ARM or x86).
+  - Windows is supported via WSL.
+
+#### Runtime Libraries
+ExecuTorch provides a native C++ runtime API, as well as platform and language-specific bindings for Python, Android, and iOS.
+
+##### Python
+Python runtime bindings are provided as part of the executorch pip package.
```

Comment:
"Python runtime bindings are provided as part of the executorch pip package. Pybindings is particularly useful during development phase for experimentation and correctness testing"